imagetest/fixtures_test.go

```diff
@@ -236,6 +237,7 @@ func TestCreateNetworkDependencies(t *testing.T) {
 	}
 	var found bool
 	for _, dep := range deps {
+		fmt.Print(dep)
```

Comment:
I'm assuming this was for debugging and not meant to be committed?

Reply:
oops. revert